MainRepository/Program.cs

```diff
@@ -1,7 +1,30 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find guess number from 1 to 127.
+namespace Test2 {
+   class GuessNumber {
+      public static void GuessGame () {
+         int n = 2;
+         List<int> BitList = new List<int> () { 0, 0, 0, 0, 0, 0, 0 };
+         List<int> Powers = new List<int> () { 64, 32, 16, 8, 4, 2, 1 };

```

Comment:
As the collection have constant length, you should use Array. Also, variable name should start with small letters.